evalml/model_understanding/prediction_explanations/_user_interface.py

```diff
@@ -423,3 +451,25 @@ def make_dict(self, data):
                                                                  data.input_features)["explanations"]
             report.append(section)
         return {"explanations": report}
+
+    def make_dataframe(self, data):
+        report = []
+        for rank, index in enumerate(data.index_list):
+            shap_table = self.table_maker.make_dataframe(index, data.pipeline, data.input_features)
+            shap_table['rank'] = rank
+            if self.make_predicted_values_maker:
+                heading = self.make_predicted_values_maker.make_dataframe(index, data.y_pred, data.y_true, data.errors,
+                                                                          pd.Series(data.input_features.index))
+                for key, value in heading.items():
+                    if key == "probabilities":
+                        for class_name, probability in value.items():
+                            shap_table[f'label_{class_name}_probability'] = probability
```

Comment:
Can we replace this `for` with
```python
labels = [f'label_{class_name}_probability' for class_name in value.keys()]
shap_table[labels] = value.values()
```
I guess that code would rely on `value.values()` being ordered in the same way as `value.keys()`, and I don't see this set up as an `OrderedDict`, so maybe what you have is fine :)

Reply:
Yea I think we can add another for-comprehension to make sure the order is the same but at that point the main benefit of that implementation over this one is saving an indent. So I think I'll stick to this one!